netlib-test/create_new_testdir.sh

```diff
@@ -131,6 +131,9 @@ main()
 	sed -i 's/zchkaa.o: zchkaa.F/zchkaa.o: zchkaa.F $(LAPACKLIB) $(BLASLIB)/' ${testdir_new}/TESTING/LIN/Makefile
 	sed -i 's/$(FC) $(FFLAGS_DRV) -c -o $@ $</$(FC) $(FFLAGS_DRV) -c -o $@ $^/g' ${testdir_new}/TESTING/LIN/Makefile
 
+	# A small fix so this script will parse correctly
+	sed -i 's/find(\"run/find(\"run\)/g' ./${testdir_new}/lapack_testing.py
+
```

Comment:
A small charge to the script. It was erroring out after all the tests had been run and it was trying to collect the data

Reply:
Fixed in https://github.com/iotamudelta/libflame/pull/10/commits/0c0c07281e4b5c4d9f82bfe6d5fb17ab0b84f2ce